src/Illuminate/Validation/Rules/Unique.php

```diff
@@ -46,7 +67,7 @@ public function __toString()
             $this->table,
             $this->column,
             $this->ignore ? '"'.$this->ignore.'"' : 'NULL',
-            $this->idColumn,
+            $this->idColumn ?? 'NULL',
```

Comment:
Can you explain why this is needed?

Reply:
because we removed the default value for the protected $idColumn; and the __toString() need string conversion for the variables